test/lib/code_corps_web/views/task_list_view_test.exs

```diff
@@ -7,7 +7,7 @@ defmodule CodeCorpsWeb.TaskListViewTest do
     task = insert(:task, order: 1000, task_list: task_list)
 
     task_list = CodeCorpsWeb.TaskListController.preload(task_list)
-    rendered_json =  render(CodeCorpsWeb.TaskListView, "show.json-api", data: task_list)
+    rendered_json =  render(CodeCorpsWeb.TaskListView, "show.json-api", %{data: task_list, conn: %Plug.Conn{}, params: task_list.id})
 
     expected_json = %{
```

Comment:
@begedin this is failing.  I returns a map with atom keys.  Should the json be a map with string keys or doesn't it matter?